src/main/java/az/edu/strangers/service/FamilyServiceImpl.java

```diff
@@ -53,7 +54,9 @@ public List<Family> getFamiliesLessThan(Integer number) {
 
     @Override
     public Long countFamiliesWithMemberNumber(Integer number) {
-        return getAllFamilies().stream()
+        return familyDao.getAllFamilies()
+                .stream()
+                .peek(System.out::println)
```

Comment:
why printing families inside this method?

Reply:
i forgot to remove that lol